plugin.xml

```diff
@@ -23,20 +23,34 @@
             <uses-permission android:name="android.permission.MANAGE_OWN_CALLS"/>
             <uses-permission android:name="android.permission.READ_PHONE_NUMBERS"/>
             <uses-permission android:name="android.permission.READ_PHONE_STATE"/>
```

Comment:
nit: Please remove all the permissions we no longer need, from both facetalk and the plugin.

Reply:
yep, left that to the end, was able to remove READ_PHONE_STATE, but we need to keep READ_PHONE_NUMBERS + CALL_PHONE.